mir-module/src/main/resources/xsl/mir-utils.xsl

```diff
@@ -31,6 +30,8 @@
     <xsl:template name="iconLink">
         <xsl:param name="baseURL"/>
         <xsl:param name="mimeType"/>
+        <xsl:param name="derivateMaindoc"/>
+        <xsl:param name="returnId"/>
```

Comment:
The parameter `returnId` is not used. Possibly a remnant of an earlier version?

Reply:
it is used in line 106